docs/api.md

```diff
@@ -571,6 +574,157 @@ Content:
 
 last are set by `/lectures/:lecture-id/engagements?last=true`
 
+### <a name="login"></a>Login
+
+Login as a lecturer
+
+#### URL
+
+`/auth/login`
+
+#### Method
+
+POST
+
+#### Data Parameters
+
+##### Required
+
+* `email=[string]` Lectures registered email address.
+* `password=[string]` Hashed password using SHA256.
+
+#### Success Responses
+
+##### Success
+
+Code: 200
+
+Content:
+```
+{
+    "type": "boolean"
+    "description": "Flag for success",
+}
+```
+
+#### Error Responses
+
+##### Wrong user credentials
+
+Either the email or password or both was wrong.
+See error message for details.
+
+Code: 404
```

Comment:
Is this the appropriate code? I think 401 seems more correct, but then there is the issue of the required WWW-Authenticate header field.